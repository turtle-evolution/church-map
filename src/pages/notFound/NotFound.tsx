const NotFound = (): JSX.Element => {
  return (
    <div data-testid="page-not-found" className="page-not-found">
      No Component Defined
    </div>
  );
};

export default NotFound;
