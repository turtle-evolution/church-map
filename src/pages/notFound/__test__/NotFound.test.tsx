import React from 'react';
import { render, screen } from '@testing-library/react';

import NotFound from '../NotFound';

describe('NotFound', () => {
  test('should take a snapshot', () => {
    const { asFragment } = render(<NotFound />);
    expect(asFragment()).toMatchSnapshot();
  });

  test('renders text is  No Component Defined', () => {
    render(<NotFound />);
    expect(screen.getByTestId('page-not-found')).toHaveTextContent(
      'No Component Defined'
    );
  });
});
