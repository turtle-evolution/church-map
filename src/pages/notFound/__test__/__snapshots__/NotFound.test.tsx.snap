// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="page-not-found"
    data-testid="page-not-found"
  >
    No Component Defined
  </div>
</DocumentFragment>
`;
